### PR TITLE
v2v: Add case for --print-estimate

### PR DESCRIPTION
**Pull request by xiaodwan**
For case: RHEL-151757

Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by xiaodwan** [comment]
(1/1) type_specific.io-github-autotest-libvirt.v2v_options.positive_test.print_estimate.default.input_mode.libvirt.esx.esx_67.default.output_mode.none: PASS (1646.99 s)
(1/1) type_specific.io-github-autotest-libvirt.v2v_options.positive_test.print_estimate.to_file.input_mode.libvirt.esx.esx_67.default.output_mode.none: PASS (1660.92 s)

**Comment by xiaodwan** [comment]
@chloerh @chunfuwen please help to review it, thanks~

File: v2v/tests/src/v2v_options.py [hunk 1]
```diff
@@ -394,6 +394,22 @@ def check_man_page(in_man, not_in_man):
...
+        Check disk size and total size in file of estimate created by v2v
+        """
+        import json
+        with open(estimate_file) as fp:
+            content = json.load(fp)
+        logging.debug('json file content:\n%s' % content)
+
+        total_size = 0
+        for i in range(len(content['disks'])):
+            total_size += content['disks'][i]
```
Comment (chloerh) [inline v2v/tests/src/v2v_options.py]: Looks like this is equal to total_size = sum(content['disks'])

Comment (xiaodwan) [reply v2v/tests/src/v2v_options.py]: Thanks, I've updated it.

**Pull request merged**